api/user.js

```diff
@@ -0,0 +1,14 @@
+import { client } from "./axiosClient"
+
+export const getUser = async (email) => {
+  return client.get(`/users?email=${email}`).then(response => response.data[0]);
+}
+
+export const createUser = async (email, name) => {
+  return client.post(`/users`, {
+    email,
+    name,
+    // username: '',
+    // phone: '',
```

Comment:
The `createUser` function has commented-out fields `username` and `phone`. If these fields are required or optional for user creation, consider uncommenting them and providing appropriate values. Otherwise, ensure that their omission is intentional.